.github/workflows/build-test-and-publish.yml

```diff
@@ -0,0 +1,188 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+      - DEV-100_deployment-on-aws
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build Elixir project
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set version number
+        id: version
+        uses: DEalog/version-action@v0.1.0
+        with:
+          fallback-prefix: "1.0.0-SNAPSHOT-"
+
+      - name: Use cached node_modules
+        id: cache
+        uses: actions/cache@v1
+        with:
+          path: ./assets/node_modules
+          key: nodeModules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            nodeModules-
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
```

Comment:
This is ATM only needed for building the final container image. Currently there are no JS tests or dependencies.